<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtUi</class>
 <widget class="QMainWindow" name="QtUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shadow Rig</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="Header" name="header">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SHADOW RIG</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QComboBox" name="cmb_mode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Rig behavior mode. Stretch: plane scales/translates without rotation. Orbit: plane rotates to face away from light.</string>
         </property>
         <item>
          <property name="text">
           <string>Mode: Stretch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mode: Orbit</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_combine">
         <property name="toolTip">
          <string>If checked, creates a single shadow for the selected object and all its descendants.</string>
         </property>
         <property name="text">
          <string>Include Children</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_source">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Name for the shadow source locator. Use different names to create separate sources.</string>
         </property>
         <property name="text">
          <string>shadow_source</string>
         </property>
         <property name="placeholderText">
          <string>Source Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="s000">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Resolution of the generated silhouette texture.</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Resolution: 512</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Resolution: 128</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Resolution: 256</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Resolution: 1024</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Resolution: 2048</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Resolution: 4096</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Resolution: 8096</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb000">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Projection axis for the silhouette.</string>
         </property>
         <item>
          <property name="text">
           <string>Auto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>X (Side)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y (Top)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z (Front)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b001">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Reset to Defaults</string>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b000">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Create Shadow</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Header</class>
   <extends>QLabel</extends>
   <header>uitk.widgets.header.Header.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
